### ISSUE TITLE
TernausNet: U-Net with VGG11 Encoder Pre-Trained on ImageNet for Image Segmentation

### ISSUE THREAD

**Issue by shuto-keio** [shuto-keio/paper_summary#2]
# INFO
## author
Vladvimir Igloviko, Alexey Shvets
## affiliation
Lyft Inc. , Massachusetts Institute of Technology
## conference or year
arxiv 2018.01
## link
[pdf](https://arxiv.org/pdf/1801.05746.pdf)

# 概要
U-Netにpre-trainedなweightを使用することで性能を向上させた。
[kaggle 「Carvana Image Masking Challenge」](https://www.kaggle.com/c/carvana-image-masking-challenge/overview)において、１位を獲得したモデル
![image](https://user-images.githubusercontent.com/38306927/57067312-9b59d600-6d09-11e9-8322-e7f457e34ce0.png)

# 提案手法
### ネットワーク
![image](https://user-images.githubusercontent.com/38306927/57069415-897b3180-6d0f-11e9-9234-d99abd209223.png)

![image](https://user-images.githubusercontent.com/38306927/57067833-1079db00-6d0b-11e9-913a-0d1fb39bad20.png)

### Loss
![image](https://user-images.githubusercontent.com/38306927/57068945-21781b80-6d0e-11e9-9b0f-1dd10bada8c0.png)
![image](https://user-images.githubusercontent.com/38306927/57068951-29d05680-6d0e-11e9-924f-08494b09d4de.png)
![image](https://user-images.githubusercontent.com/38306927/57068979-32289180-6d0e-11e9-8ec0-f4f5511b213d.png)
![image](https://user-images.githubusercontent.com/38306927/57068990-3fde1700-6d0e-11e9-999c-c3a03b70a5a8.png)

# 検証
### dataset
Inria Aerial Image Labeling Dataset
街の航空写真のデータセット、建物かそうでないかを0,1で表現

(B)LeCunの初期値
(C)VGG11 pretrained model
(Encoder:Vgg11 Decoder:LeCun)
(D)full network trained on the Carvana dataset
![image](https://user-images.githubusercontent.com/38306927/57069271-140f6100-6d0f-11e9-821c-63003b93b322.png)

# 新規性
U-Netにおいて、VGGの部分の重みをpretrainすることで性能の向上

# 議論，展望
特になし

# date
2019/05/02
